### DISCUSSION TITLE
PrimeVue Calendar Time Picker onBlur bug

### DISCUSSION THREAD

**Discussion by craiggeil** [primefaces/community#275] [PrimeVue]
In the panel dropdown, when you click anywhere on the panel that is outside the up and down arrows of the hours and minutes the blur event is getting called. Even clicking the AM/PM arrows causes a blur event to occur. 

https://stackblitz.com/edit/bveubz?file=src%2FApp.vue%3AL31,package.json

**Comment by tugcekucukoglu** [comment]
Thanks for the reproducer demo. blur event invokes every time when input loses its focus. For more definitions https://primevue.org/calendar/#api.calendar.emits.blur

**Comment by craiggeil** [comment]
Well that isn't happening if you read the post again and try out the example.

**Comment by tugcekucukoglu** [comment]
When you click anywhere outside the input, the input will lose focus and the blur event will be called. This can be a button anywhere on the panel.

**Comment by craiggeil** [comment]
_Even clicking the AM/PM arrows causes a blur event to occur._
Also when you display a panel with inputs on them the panel should not cause a blur, that seems kind of obvious but maybe not.
